acsoo/templates/project/+project.name+/.gitlab-ci.yml.bob

```diff
@@ -16,6 +16,10 @@ before_script:
         ssh-add <(echo "$SSH_DEPLOY_KEY")
         echo "${SSH_AGENT_PID}" > ssh-agent.pid
     }
+    function makepot {
+        venv/bin/click-odoo-makepot -c odoo-ci.cfg -d ${DB_NAME} --commit --msgmerge --addons-dir=$(pwd)/odoo/addons
+        git push "git@gitlab.acsone.eu:${CI_PROJECT_NAMESPACE}/${CI_PROJECT_NAME}.git" "HEAD:${CI_COMMIT_REF_NAME}"
```

Comment:
@sbidoul  `${CI_PROJECT_NAMESPACE}/${CI_PROJECT_NAME}` could be written: `${CI_PROJECT_PATH}`
https://docs.gitlab.com/ee/ci/variables/#predefined-variables-environment-variables

Reply:
@ThomasBinsfeld done, tested.